doc/node.1

```diff
@@ -398,6 +398,13 @@ See
 and
 .Ev SSL_CERT_FILE .
 .
+.It Fl -use-largepages Ns = Ns Ar num
+Re-map the Node.js static code to large memory pages at startup. If supported on
+the target system, this will cause the Node.js static code to be moved onto 2
+MiB pages instead of 4 KiB pages.
+.Pp
+The following values are valid: 0 (do not map), 1 (map, ignore failure, and do not report it), and 2 (map, ignore failure, but report it to standard error).
```

Comment:
Is there a particular reason that we use numbers here instead of some enum-like strings?

Reply:
I was thinking that they are faster to compare, so as not to impact startup time. If you don't believe that's a good enough reason, I'm happy to change the values to strings. The precedent for enum-like strings is `--unhandled-rejections`, which accepts `"strict"`, `"warn"`, or `"none"`.

Reply:
Compared to all the things that we need to do during startup, the overhead of comparing strings with only 3 possible options seem to be negligible to me, so I think yes, it's better to have more readable options here.

Reply:
I’d also prefer a string here (and this line should be wrapped at 80 characters :))